Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpBoneImporter.cpp

```diff
@@ -106,37 +67,39 @@ namespace AZ
                     return Events::ProcessingResult::Ignored;
                 }
 
-                bool isBone = false;
-                
-                {
-                    AZStd::unordered_map<AZStd::string, const aiBone*> boneLookup;
-                    MakeBoneMap(scene, boneLookup);
+                AZStd::unordered_multimap<AZStd::string, const aiBone*> boneByNameMap;
```

Comment:
We'll soon replace that whole part with https://github.com/assimp/assimp/blob/master/code/PostProcessing/ArmaturePopulate.h
This will require us to recompile the AssImp lib though.